tools/EteriTools/CollectResults/evaluation.py

```diff
@@ -0,0 +1,114 @@
+import glob, os, sys
+import shutil
+from shutil import copyfile
+from os import system
+import re
+#import sklearn
+#from sklearn import metrics
+#from tabulate import tabulate
+
+
+
+def sh(script):
+    system("bash -c '%s'" % script)
+
+#dataNames = "test-data/FASTA/data.names"
+dataNames = "FASTA/data.names"
```

Comment:
here this is hardcoded because i pass FASTA folder to the tool and there all the names are defined by myself in 'preprocessing' step.